plugins/ufm_consumer_plugin/README.md

```diff
@@ -0,0 +1,148 @@
+# UFM Plugins
```

Comment:
The README file should be updated with the information that related to the UFM Consumer plugin and the instructions on how to deploy it

Reply:
you can take a look as an example to any of the existing plugins (e.g. https://github.com/Mellanox/ufm_sdk_3.0/tree/main/plugins/bright_plugin)

Reply:
README I will update a bit later

Reply:
Alex, This README is not relevant to the UFM Consumer Plugin - it is a generic README for all the plugins,
Need to update this README with UFM Consumer Plugin related information - deployment, configuration and lifecycle management - enabling / disabling it.

Reply:
I answered to Anan. Nothing to update. 
Need to do it.